web/src/pages/studies/index.tsx

```diff
@@ -8,9 +12,23 @@ export default function StudiesPage() {
         title="Studies | ARC Services Portal"
         description="View and modify studies in the ARC Services Portal"
       />
-      <div>
-        <Studies />
-      </div>
+      <Title text={"Studies"} />
+      <p className={styles.subtitle}>
+        <span className={styles.highlighted}>Studies</span> are a top level entity. They can contain{" "}
+        <strong className={styles.coloured}>projects</strong>{" "}
+        <InfoIcon title="Projects are owned by a study and can contain assets" className={styles.info} /> and{" "}
```

Comment:
these don't hover for me 😕

Reply:
Have made a custom component for now, maybe once they've sorted out their tooltip stuff we can convert